wolfcrypt/src/pkcs7.c

```diff
@@ -13098,8 +13115,10 @@ int wc_PKCS7_EncodeAuthEnvelopedData(wc_PKCS7* pkcs7, byte* output,
     if (unauthAttribsSz > 0) {
         XMEMCPY(output + idx, unauthAttribSet, unauthAttribsSetSz);
         idx += (int)unauthAttribsSetSz;
-        XMEMCPY(output + idx, flatUnauthAttribs, unauthAttribsSz);
-        idx += (int)unauthAttribsSz;
+        if (unauthAttribsSz > 0) {
```

Comment:
is this the same check as whats on line 13115? if so maybe the first needs checked on unauthAttribsSetSz and this new one as is with unauthAttribsSz

Reply:
good catch.  was just a benign flub, duplicate check to prevent null deref.  `clang-tidy`  wasn't actually warning on the `XMEMCPY` there, and now I see that's because @cconlon already had it properly guarded.